torch_xla/csrc/tensor.cpp

```diff
@@ -607,6 +607,11 @@ bool XLATensor::ShouldSyncIrNode() {
   return this->data()->ir_value->op() != xla_device_data;
 }
 
+bool XLASymNodeImpl::is_bool() {
```

Comment:
I actually don't know what's the right thing to do so. Just try a few different values and this one works.

Reply:
@ezyang Can you shed some light on the expected behavior on this function?(along with `is_int` and `guard_bool`)

Reply:
So the intended semantics is, when we do an equality operation, the resulting SymNode should be treated as a bool. You'll have to add some sort of internal tagging to XLA to do this, I think.

Reply:
Thanks, Ed. Implemented corresponding changes.